test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class VotesControllerTest < ActionController::TestCase
+  test "vote on post" do
+    user = FactoryGirl.create(:user)
+    post = FactoryGirl.create(:post, user: user)
```

Comment:
This assigns the variable `post` to be a post from the database.  This means `post` will not trigger the `post` method, instead it will reference this variable.